### ISSUE TITLE
message with two different registration_id arrive to the same device

### ISSUE THREAD

**Issue by StErMi** [xtrinch/fcm-django#169]
Hi there, I just finished set upping locally the project and I've added 2 different registration_id, one from an iPhone and one from Android.

The mobile app is a capacitor app, I've configured it correctly and those registration_id come from the app.
The strange thing is that if I send the message to both of them, I get 2 pushes on the iPhone. If I disable the iPhone registration token, the Android gets one notification.

What could that be?

This is the code I'm using to send the notification:

`FCMDevice.objects.send_message(Message(notification=Notification(title="test ios3", body="test body3")))`

**Comment by StErMi** [comment]
It's strange, maybe I didn't get how to properly use it. If I send those messages one per device it works (1 notification sent to each device)

```
        devices = FCMDevice.objects.all()
        for device in devices:
            single_result = device.send_message(
                Message(
                    notification=Notification(title="test ios4", body="test body4"),
                    data={
                        "title": "test ios5",
                        "body": "test body5",
                        "view": "DEVICE",
                        "id": "4"
                    },
                )
            )
 ```

what am I doing wrong?

**Comment by jordandjp** [comment]
Hi StErMi, your implementation is not wrong, I also had this issue and checking I found a bug when sending multiple messages and made a merge request ([172](https://github.com/xtrinch/fcm-django/pull/172)) to fix it.

In the meantime, you could iterate and send single messages like you did, or else is to create a proxy model to override the method with the changes in MR and use this model instead of the original.

**Comment by xtrinch** [comment]
Fixed in 1.0.4
